client/ayon_maya/plugins/publish/collect_ornatrix_rig.py

```diff
@@ -0,0 +1,82 @@
+import os
+from typing import List, Dict, Any
+import pyblish.api
+from ayon_core.pipeline.publish import KnownPublishError
+from ayon_maya.api import lib
+from ayon_maya.api import plugin
+from maya import cmds
+
+
+ORNATRIX_NODES = {
+    "HairFromGuidesNode", "GuidesFromMeshNode",
+    "MeshFromStrandsNode", "SurfaceCombNode"
+}
+
+
+class CollectOxRig(plugin.MayaInstancePlugin):
+    """Collect all information of the Ornatrix Rig"""
+
+    order = pyblish.api.CollectorOrder + 0.4
+    label = "Collect Ornatrix Rig"
+    families = ["oxrig"]
+
+    def process(self, instance):
+        ornatrix_nodes = cmds.ls(instance.data["setMembers"], long=True)
+        self.log.debug(f"Getting ornatrix nodes: {ornatrix_nodes}")
+        # Force frame range for yeti cache export for the rig
+        # Collect any textures if used
+        ornatrix_resources = []
+        for node in ornatrix_nodes:
+            # Get Yeti resources (textures)
+            resources = self.get_texture_resources(node)
+            ornatrix_resources.extend(resources)
+        # avoid duplicate dictionary data
+        instance.data["resources"] = [
+            i for n, i in enumerate(ornatrix_resources)
+            if i not in ornatrix_resources[n + 1:]
+        ]
+        self.log.debug("{}".format(instance.data["resources"]))
```

Comment:
Can't this just be this?
```suggestion
        self.log.debug(instance.data["resources"])
```